src/vinyldns/zone.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2018 Comcast Cable Communications Management, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add module docstring."""
+
+
+class ACLRule(object):
+    def __init__(self, access_level, description, user_id, group_id, record_mask, record_types):
```

Comment:
Should `description` at least be optional with defaults? `user_id`, `group_id` and `record_mask` are also optional in the API.

Reply:
Correct, I can do that

Reply:
Also, I will add an enum for `access_level`